code/modules/library/skill_learning/skillchip.dm

```diff
@@ -96,3 +98,23 @@
 	skill_icon = "spa"
 	implanting_message = "<span class='notice'>Your mind is filled with plant arrangments.</span>"
 	removal_message = "<span class='notice'>Your can't remember how a hedge looks like anymore.</span>"
+
+/obj/item/skillchip/useless_adapter
+	name = "Skillchip adapter"
+	skill_name = "Useless adapter"
+	skill_description = "Allows you to insert another identical skillchip into this adapter, but the adapter also takes a slot ..."
+	skill_icon = "plug"
+	implanting_message = "<span class='notice'>You can now implant another chip into this adapter, but the adapter also took up an existing slot ...</span>"
+	removal_message = "<span class='notice'>You no longer have the useless skillchip adapter.</span>"
+	allow_multiple = 1
+	slot_cost = 0
+
+/obj/item/skillchip/useless_adapter/on_apply(mob/living/carbon/user, silent)
+	user.max_skillchip_slots++
+	user.used_skillchip_slots++
+	return ..()
```

Comment:
```suggestion
/obj/item/skillchip/useless_adapter/on_apply(mob/living/carbon/user, silent)
	. = ..()
	user.max_skillchip_slots++
	user.used_skillchip_slots++
```
Try to call parent first on addition and last on removal, so we wrap from the parent-down and unwrap from the child-up.

Reply:
Yeah I see now what you wanted to tell me with your first suggestion.